### DISCUSSION TITLE
Multiline support for OpenTofu Console

### DISCUSSION THREAD

**Discussion by boldandbusted** [opentofu/opentofu#1300] [Ideas]
Howdy. I posted this feature request at the Terraform Issues page. It got 57 upvotes! :) Perhaps this is something the OpenTofu crew could take on, and impliment it before the BSL-encumbered project does? Others have also chimed in with some refinement ideas, too. Cheers!

https://github.com/hashicorp/terraform/issues/32954

**Comment by ghost** [comment]
Hewy @boldandbusted could you please raise this as an issue so the core team can properly discuss and address it? Thank you for your input!

**Reply by boldandbusted** [reply]
Done! https://github.com/opentofu/opentofu/issues/1307 :)
